app/views/devise/sessions/new.html.erb

```diff
@@ -86,4 +86,5 @@
   </p>
 <% end %>
 
+<%= render 'shared/dap_analytics' if IdentityConfig.store.participate_in_dap %>
```

Comment:
This will render in the `<main>` tag rather than at the end of `<body>`.

I'll try to see if I can accommodate third-party / fully-qualified URL scripts into our existing script helpers.

Reply:
> I'll try to see if I can accommodate third-party / fully-qualified URL scripts into our existing script helpers.

Added support in e205f10 and updated rendering in 030a80f..